### ISSUE TITLE
Remove extra dependency for deoplete-solargraph.py

### ISSUE THREAD

**Issue by uplus** [uplus/deoplete-solargraph#6]
I separated `deoplete-solargraph` and` solargraph-utils.py` from the viewpoint of maintainability and modularity.
This made development very easy, but for users it would be troublesome to add more packages to manage.

But, if `solargraph-utils.py` is merged, dependencies on` solargraph` and python `neovim-client` remain.
So, I don't feel much merit in merging `deoplete-solargraph.py`.

The best solution I think is to make solargraph-utils.py manageable with vim's plugin manager.

**Comment by iovis9** [comment]
But solargraph is used by other editors for autocompletion and neovim client is used by other plugins in neovim. `deoplete-solargraph.py` is only used by this plugin, that's why I find it troublesome. But I agree it's probably easier to maintain separately.

**Comment by halftan** [comment]
No, it doesn't make it easier to maintain this plugin and it's added extra effort to develop/bugfix this plugin since I need to open up two workspaces and switching between each other frequently. The two projects are coupling so close to each other and I highly doubt that there will be another repo depending on `solargraph-utils.py`, thus there's no benefit on separating these two repos.
